### PR TITLE
Update acq basic tutorial doc for 1488 and 1489 support

### PR DESCRIPTION
**Pull request by jon-perreault**
### What does this Pull Request accomplish?

Update the basic acquisition tutorial doc for 1488 and 1489 hardware support.

### CODE REVIEW

File: docs/tutorials/gettingstartedexample/gse-acq-basic.md [hunk 1]
```diff
@@ -15,18 +15,26 @@ This tutorial will teach you the steps needed to configure an acquisition using
...
+Review and complete all setup from the appropriate getting started guide:
+- [PXIe-1486 Getting Started Guide](https://www.ni.com/docs/en-US/bundle/pxie-1486-getting-started/)
+- [PXIe-1487 Getting Started Guide](https://www.ni.com/docs/en-US/bundle/pxie-1487-getting-started/)
+- [PXIe-1488 Getting Started Guide](https://www.ni.com/docs/en-US/bundle/pxie-1488-getting-started/)
+- [PXIe-1489 Getting Started Guide](https://www.ni.com/docs/en-US/bundle/pxie-1489-getting-started/)
 
 Basic knowledge of LabVIEW and LabVIEW FPGA concepts. 
 
-A supported interface module and camera on a PXI system running Windows.
+A supported interface module and camera on a PXI system running Windows. The table below lists all supported interface modules and supported cameras that have configuration scripts provided with the getting started example. Configuration scripts for cameras not listed in the table below need to be provided separately.
```
Comment (hernstrom) [inline docs/tutorials/gettingstartedexample/gse-acq-basic.md]: Suggest rewording for clarity at a minimum, but see the further comment below for a more comprehensive potential change:

"For cameras not listed in the table below, configuration scripts need to be provided separately."

File: docs/tutorials/gettingstartedexample/gse-acq-basic.md [hunk 2]
```diff
@@ -15,18 +15,26 @@ This tutorial will teach you the steps needed to configure an acquisition using
...
+| PXIe-1487 (8 In - 9296A Deserializer)        | [LI-IMX490-GMSL2](https://www.leopardimaging.com/product-category/autonomous-camera/maxim-gmsl2-cameras/li-imx490-gmsl2/)             |
+| PXIe-1487 (4 In 4 Out - 9295A/9296A SerDes)  | [LI-IMX490-GMSL2](https://www.leopardimaging.com/product-category/autonomous-camera/maxim-gmsl2-cameras/li-imx490-gmsl2/)             |
+| PXIe-1488 (8 In - 9702 Deserializer)         |                                                                                                                                       |
+| PXIe-1488 (4 In 4 Out - 971/9702 SerDes)     |                                                                                                                                       |
+| PXIe-1489 (4 In - 96792A Deserializer)       |                                                                                                                                       |
+| PXIe-1489 (2 In 2 Out - 96793/96792A SerDes) |                                                                                                                                       |
```
Comment (hernstrom) [inline docs/tutorials/gettingstartedexample/gse-acq-basic.md]: Why are we listing blank rows in the table?  Is it just to convey that this module doesn't have a script that is provided?

If scripts are coming soon, then blank rows seems ok, but if nota nother option, remove the blank rows and change the last sentence above the table to read:

"For modules & cameras not listed in the table below, configuration scripts need to be provided separately."

**Review by hernstrom — CHANGES_REQUESTED** [review]

File: docs/tutorials/gettingstartedexample/gse-acq-basic.md (hunk 1 above)
Comment (jon-perreault) [reply docs/tutorials/gettingstartedexample/gse-acq-basic.md]: Done.

File: docs/tutorials/gettingstartedexample/gse-acq-basic.md (hunk 2 above)
Comment (jon-perreault) [reply docs/tutorials/gettingstartedexample/gse-acq-basic.md]: I added the suggested wording change and removed the blank rows from the table.

**Review by hernstrom — APPROVED** [review]

**Pull request merged**